src/test/java/build/buildfarm/instance/shard/RedisShardBackplaneTest.java

```diff
@@ -100,6 +100,10 @@ void verifyChangePublished(JedisCluster jedis) throws IOException {
     assertThat(opChange.getReset().getOperation().getName()).isEqualTo("op");
   }
 
+  String OperationName(String name) {
```

Comment:
nit: dislike the methods-with-proper-name capitalization

Reply:
fixed